homeassistant/components/file/config_flow.py

```diff
@@ -79,8 +78,7 @@ async def async_step_notify(
             if not await self.validate_file_path(user_input[CONF_FILE_PATH]):
                 errors[CONF_FILE_PATH] = "not_allowed"
             else:
-                name: str = user_input.get(CONF_NAME, DEFAULT_NAME)
-                title = f"{name} [{user_input[CONF_FILE_PATH]}]"
+                title = f"{DEFAULT_NAME} [{user_input[CONF_FILE_PATH]}]"
```

Comment:
A future improvement could be to make a helper method that both the notify and sensor steps can use since most of the logic is the same in those steps.

Reply:
I'll open a follow up PR for this

Reply:
https://github.com/home-assistant/core/pull/117371